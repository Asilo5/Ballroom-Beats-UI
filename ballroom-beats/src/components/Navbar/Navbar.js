import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import { createStackNavigator } from 'react-navigation-stack';
import { withNavigation } from 'react-navigation';

class Navbar extends Component {

  render() {
      return (
          <View style={styles.container}>
            <Text>Navbar</Text>
          </View>
      )      
  }
};  

const AppNavigator = createStackNavigator({
    Navbar: {
      screen: Navbar,
    },
});
  
const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: 'white',
        alignItems: 'center',
        justifyContent: 'center',
    }
});

export default withNavigation(Navbar);