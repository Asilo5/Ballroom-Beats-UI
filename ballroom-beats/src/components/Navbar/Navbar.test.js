import React from 'react';
import Navbar from '../Navbar/Navbar';
import { shallow } from 'enzyme';

describe('Navbar', () => {

   it('should match snapshot', () => {
     const wrapper = shallow(<Navbar />);
     expect(wrapper).toMatchSnapshot();
   });

});