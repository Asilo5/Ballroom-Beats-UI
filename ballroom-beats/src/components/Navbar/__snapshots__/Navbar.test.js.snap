// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#D1005A",
      "bottom": 1,
      "position": "absolute",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "#007AFF",
            "borderRadius": 5,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "flex-start",
              "padding": 8,
            },
            Object {
              "backgroundColor": "#007AFF",
              "borderRadius": 5,
            },
            Object {
              "backgroundColor": "#D1005A",
              "fontSize": 30,
              "padding": 20,
            },
          ]
        }
        title="BACK"
      >
        <Text />
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 30,
              "fontWeight": "bold",
              "textDecorationLine": "none",
              "textShadowColor": "#000001",
              "textShadowOffset": Object {
                "height": 2,
                "width": 2,
              },
              "textShadowRadius": 2,
            }
          }
        >
          Home
        </Text>
      </View>
    </View>
  </View>
</View>
`;
